bin/plpbuild

```diff
@@ -88,7 +95,7 @@ parser = argparse.ArgumentParser(
 parser.add_argument('command', metavar='CMD', type=str, nargs='*',
                    help='a command to be executed')
 
-parser.add_argument("--threads", dest="threads", default=1, type=int, help="Specify the number of worker threads")
+parser.add_argument("--threads", dest="threads", default=def_threads(), type=int, help="Specify the number of worker threads")
```

Comment:
This feature for parallel compilation is not yet stable, it cannot be activated by default

Reply:
That's "unfortunate". What are the symptoms if it fails? Is there a public issue tracking this? At the very least this problem should documented in the usage message of the option.